src/devices/wifi/nm-iwd-manager.c

```diff
@@ -188,8 +203,50 @@ psk_agent_export (GDBusConnection *connection, gpointer user_data,
 		(GDBusArgInfo **) &request_passphrase_out_args,
 		NULL,
 	};
+	static const GDBusMethodInfo request_private_key_passphrase_info = {
+		-1,
+		(gchar *) "RequestPrivateKeyPassphrase",
+		(GDBusArgInfo **) &request_passphrase_in_args,
+		(GDBusArgInfo **) &request_passphrase_out_args,
+		NULL,
+	};
+	static const GDBusArgInfo *const request_user_name_and_password_in_args[] = {
+		&agent_request_arg_network,
+		NULL,
+	};
+	static const GDBusArgInfo *const request_user_name_and_password_out_args[] = {
+		&agent_request_arg_user,
+		&agent_request_arg_password,
+		NULL,
+	};
+	static const GDBusMethodInfo request_user_name_and_password_info = {
+		-1,
+		(gchar *) "RequestUserNameAndPassword",
+		(GDBusArgInfo **) &request_user_name_and_password_in_args,
+		(GDBusArgInfo **) &request_user_name_and_password_out_args,
+		NULL,
+	};
+	static const GDBusArgInfo *const request_user_password_in_args[] = {
+		&agent_request_arg_network,
+		&agent_request_arg_user,
+		NULL,
+	};
+	static const GDBusArgInfo *const request_user_password_out_args[] = {
+		&agent_request_arg_password,
+		NULL,
+	};
+	static const GDBusMethodInfo request_user_password_info = {
```

Comment:
could you rework these to use our macros ([[1]](https://cgit.freedesktop.org/NetworkManager/NetworkManager/tree/src/settings/plugins/ifcfg-rh/nms-ifcfg-rh-plugin.c?id=f4bef9a12eba69e2cd94b2428939d2a523f4fce0#n824), [[2]](https://cgit.freedesktop.org/NetworkManager/NetworkManager/tree/src/nm-dbus-manager.c?id=f4bef9a12eba69e2cd94b2428939d2a523f4fce0#n1455))?

Reply:
Done too, it looks better now.